kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -429,7 +428,13 @@ def _retrieve_stats(self):
             return {}
 
     def _report_node_metrics(self, instance_tags):
-        node_spec = self._retrieve_node_spec()
+        try:
+            node_spec = self._retrieve_node_spec()
+        except requests.HTTPError:
```

Comment:
Do you need to check a specific status here?

Reply:
The HTTPError didn't contain the return_code. I need to change the signature of the `self._retrieve_node_spec()` to return the `requests.Response` instead of the json payload

Reply:
@xornivore  PR updated